core/src/modules/configLoaders/index.ts

```diff
@@ -111,6 +112,12 @@ export async function processConfigObjects(
 
   configObjects = sortConfigurationObjects(configObjects);
 
+  const { errors: validationErrors } = validateConfigObjects(configObjects);
+  validationErrors.map((err) =>
+    log(`[ config ] ${err}`, env === "test" ? "info" : "error")
+  );
+  errors.push(...validationErrors);
+
   for (const i in configObjects) {
```

Comment:
perhaps `for (const i in configObjects && errors.length === 0) {` to not continue if there were duplicates?  Otherwise, you might overwrite something you didn't intend.